tsconfig.json

```diff
@@ -13,7 +13,13 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "jsx": "preserve",
-    "incremental": true
+    "incremental": true,
+    "baseUrl": ".",
+    "paths": {
+      "@/styles/*": ["styles/*"],
```

Comment:
I believe that this should be `@styles/*` instead, we can remove the backslash, anyways, this folder is barely used so I think we can omit this alias

Reply:
ok, I will remove